installation/docker/image_override.env

```diff
@@ -0,0 +1,5 @@
+APIServerContainerImage=docker.io/tehsmash/vmclarity-apiserver:microservices
```

Comment:
`docker.io/tehsmash` - do you want to keep it?

Reply:
This is just an example of how to override the images to a different version its not used by default so I think its ok, WDYT?